nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -311,6 +315,15 @@ private Handler loadInitialWars(final Set<Bundle> bundles) {
         return gzip(webAppContextHandlers);
     }
 
+    @Override
+    public void reloadSSLContextFactory() throws Exception {
```

Comment:
Minor adjustment, recommend renaming to follow class naming for the Jetty `SslContextFactory`:
```suggestion
    public void reloadSslContextFactory() throws Exception {
```